tests/test_fields.py

```diff
@@ -545,6 +546,14 @@ def test_healpix_to_glass_spectra():
     np.testing.assert_array_equal(out, [11, 22, 21, 33, 32, 31, 44, 43, 42, 41])
 
 
+def test_glass_to_healpix_alm():
+    inp = np.array([11, 22, 21, 33, 32, 31, 44, 43, 42, 41])
+    out = glass.fields._glass_to_healpix_alm(inp)
+    np.testing.assert_array_equal(
+        out, np.array([11, 22, 33, 44, 21, 32, 43, 31, 42, 41])
+    )
```

Comment:
Let's make this more obvious

```suggestion
    inp = np.array([00, 10, 11, 20, 21, 22, 30, 31, 32, 33])
    out = glass.fields._glass_to_healpix_alm(inp)
    np.testing.assert_array_equal(
        out, np.array([00, 10, 20, 30, 11, 21, 31, 22, 32, 33])
    )
```